homeassistant/components/litterrobot/__init__.py

```diff
@@ -16,15 +18,46 @@
     Platform.VACUUM,
 ]
 
+PLATFORMS_BY_TYPE = {
+    LitterRobot: (
+        Platform.SELECT,
+        Platform.SENSOR,
+        Platform.SWITCH,
+        Platform.VACUUM,
+    ),
```

Comment:
You probably don't need this section, but otherwise LGTM

Reply:
Guess we can do the same operation for the unload.